### ISSUE TITLE
@@@ in popup

### ISSUE THREAD

**Issue by matthewsia98** [giusgad/pets.nvim#9]
Not sure what's causing the @@@ to appear in the bottom right corner
![image](https://user-images.githubusercontent.com/71731492/219450298-a91b6adb-b780-4650-8dd0-11ea04ee2da1.png)

**Comment by giusgad** [comment]
I'm not really sure why that is happening either, and I can't reproduce the issue.
I would try setting the highlight group's foreground to `NONE` or changing the `blend` value, but it might be unrelated...
If you achieve something this way or another, please let me know! Thank you!

**Comment by matthewsia98** [comment]
I thought the problem might have been related to noice or lualine, but I tried disabling both and I still see the `@`

**Comment by GustavoKatel** [comment]
the problem is that is truncating the buffer text. From `:h display`

```
lastline	When included, as much as possible of the last line
			in a window will be displayed.  "@@@" is put in the
			last columns of the last screen line to indicate the
			rest of the line is not displayed.
```

probably because it's filling the buffer with really long strings [here](https://github.com/giusgad/pets.nvim/blob/main/lua/pets/animations.lua#L8-L15). Is there a better way to do this?

**Comment by matthewsia98** [comment]
If you can recreate the issue, can you check if [this](https://github.com/giusgad/pets.nvim/pull/13) fixes it?

**Comment by giusgad** [comment]
I've merged your pr #13 @matthewsia98. Can you confirm the issue is resolved?

**Comment by matthewsia98** [comment]
Confirm fixed by #13.
